.github/dead_link_check_config.json

```diff
@@ -11,6 +11,9 @@
     },
     {
       "pattern": "^https://console.cloud.tencent.com/cos/bucket"
+    },
+    {
+      "pattern": "^#"
```

Comment:
What does this "^#" mean?

Reply:
It's a temporary solution, ignore internal link/anchor check.
[tcort/markdown-link-check@latest](https://github.com/tcort/markdown-link-check/releases/tag/v3.10.2) The latest release resolved checking the link in \<img src='xxx'\>. But the link checker is not able to resolve reference links and it says as dead links.Like

```markdown
# 概述
[概述](#概述) // It will be said as a dead link.
```

My test result:
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/30225423/170669432-32c3dec6-a613-443a-997a-d24fc9210935.png">

See https://github.com/gaurav-nelson/github-action-markdown-link-check/pull/129

Reply:
已经定位到问题的原因,是一个上游依赖的问题... [markdown-link-extractor v2->v3](https://github.com/tcort/markdown-link-extractor/commit/69252dc209389169475ff5ce5150984c403b1d19) 在重构的时候欠缺一些UT, v2版本的link是没有进行encodeURI的,升级到v3后所有的中文都被encode了...导致后面在判定时,中文的都会被当成了dead link...

判定方式为:
```
anchors.includes(link) ? 200 : 404

// v2.0.1
// links: [ '#foo', '#bar', '#potato', '#中文' ]
// anchors: [ '#foo', '#bar', '#baz', '#uh-oh', '#中文' ]

// v3.0.2
// links: [ '#foo', '#bar', '#potato', '#tomato', '#%E4%B8%AD%E6%96%87' ]
//anchors: [ '#foo', '#bar', '#baz', '#uh-oh', '#中文' ]
```

Reply:
Awesome!
We can submit an issue to them